frameworks/helloworld/tests/test_profile_mount_volumes.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import pytest
+
+import sdk_install
+import sdk_tasks
+from tests import config
+
+log = logging.getLogger(__name__)
+
+NUM_HELLO = 2
+
+
+@pytest.fixture(scope="module", autouse=True)
+def configure_package(configure_security):
+    try:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+        options = {
+            "service": {"yaml": "pod-profile-mount-volume"},
+            "hello": {"volume_profile": "xfs"}  # hardcoded in `tools/create_testing_volumes.py`.
+        }
+
+        sdk_install.install(config.PACKAGE_NAME, config.SERVICE_NAME, NUM_HELLO, additional_options=options)
+
+        yield  # let the test session execute
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.mark.sanity
+@pytest.mark.dcos_min_version("1.12")
+def test_profile_mount_volumes():
+    sdk_tasks.check_running(config.SERVICE_NAME, NUM_HELLO)
```

Comment:
Would it be possible to verify here that the correct `xfs` volumes were indeed consumed? e.g. by fetching and looking at state.json or slaves.json. For some examples of accessing mesos jsons, see `sdk_tasks.py`, e.g.: `sdk_cmd.cluster_request("GET", "/mesos/tasks").json()["tasks"]`

Reply:
The task itself already checked the filesystem (`df -t $VOLUME_PROFILE`). Wouldn't that be sufficient?

Reply:
Oh I missed that. I think that's fine.